src/pathfinder/follow_track.hpp

```diff
@@ -228,8 +228,6 @@ struct CFollowTrackT
 			m_is_station = true;
 		} else if (IsRoadTT() && IsRoadStopTile(m_new_tile)) {
 			m_is_station = true;
-		} else {
-			m_is_station = false;
```

Comment:
I think this change decreases readability.

Reply:
Why do we need an unnecessary and redundant assignation on the code?

Reply:
It's redundant, but it makes it obvious what the false 'case' is.

TBH I'd probably write the whole thing as `m_is_station = (IsRail...) || (IsRoad...)` anyway

Reply:
@LordAro 
If you look at the whole function from the beginning, it doesn't make sense. In fact, adding the unnecessary assignation may leave you thinking that somewhere in the middle of the function the value of m_is_station may be changed, which is not true.
About putting the whole thing as `m_is_station = (IsRail...) || (IsRoad...)`, may be a good idea but I don't like it. If at some point somebody needs a `bool m_is_depot` or something similar, having the code separated by cases is better.

Reply:
If anything this whole class should probably be updated to follow the "all member access must be explicit through `this->`" rule, if it's being updated anyway.

Reply:
@nielsmh Ok. But I would leave this-> for another PR.